### PR TITLE
Add OLS blog post

### PR DESCRIPTION
**Pull request by KristinaRiemer**
This will summarize our experiences with the Open Life Science program.

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
@dlebauer and @MagicMilly, there is a fairly revised draft of the blog post here, which could use y'all's feedback. Beyond just making sure I didn't leave anything out or there's extra stuff in here, some specific things to address are: 
- This could use a couple images, but I'm not sure of what (which makes me miss field work, it was easy to have cool pretty pictures!) 
- Replacing generic positive adjectives with more descriptive ones
- Is it too long?

File: content/post/2020-05-22-ols-results.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+This was a great opportunity for learning more about how to do this and implementing some productive changes. 
+These are summarized below, and also in [our final OLS presentation](https://github.com/az-digitalag/organization/blob/master/presentations/20200520_OLS.pdf). 
+
+We made progress on organizing the already dense and complicated documentation available for the drone processing pipeline so that it is more intuitive and usable. 
+We decided that the main website for the project should be [the OLS wiki for the pipeline](https://osf.io/xdkcy/), with the main landing page for users being the [home page](https://osf.io/xdkcy/wiki/home/). 
+This links to the code base and community health files in repos in [the GitHub organization](https://github.com/agPipeline), which then links to [more technical documentation aimed primarily at developers](https://agpipeline.github.io/). 
+
+We additionally had a chance to sit down and talk to one of the users of a similar pipeline. 
+He had insightful comments on its usability, including that having an more experienced user to talk to was crucial and that the number of technical tools one needs to be familiar with is substantial.
```
Comment (MagicMilly) [inline content/post/2020-05-22-ols-results.md]:
```suggestion
He had insightful comments on its usability, including that having a more experienced user to talk to was crucial and that the number of technical tools one needs to be familiar with is substantial. 
```

File: content/post/2020-05-22-ols-results.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+In this first cohort, there were 29 mentees with 20 mentors. 
+Throughout the program, the mentees would meet every other week with their mentor to discuss their project and next steps. 
+On opposing weeks, the mentees met collectively with the organizers. 
+These cohort meetings included lots of excellent guest speaker talks, such as [Karin Lagesen](https://folk.uio.no/karinlag/) on codes of conduct, [Alex Chan](https://alexwlchan.net/) on designing for inclusion, and [Josh Simmons](https://joshsimmons.com/) on open licensing. 
+Then to summarize and celebrate everyone's progress during the 18 weeks of the program, all of the mentees delivered short presentations, which are on [YouTube](https://www.youtube.com/c/OpenLifeSci). 
+
+The OLS program went above and beyond our expectations. 
+There was so much knowledge sharing and networking that went beyond just how to do science openly to how to build up a community around open science products. 
+We learned so much about consistently having a community-first perspective, including designing for openness and inclusivity from the beginning of a project, which we will implement in our work. 
+
```
Comment (MagicMilly) [inline content/post/2020-05-22-ols-results.md:40]: Maybe adding something about the code of conduct in this section of `community-first perspective`?

**Review by MagicMilly — APPROVED** [review]

**Comment by MagicMilly** [comment]
Not too long - I think it's great! We could add the OLS leaf logo and maybe a screenshot from the `Insights` section on GitHub repos that Katrin told us about? The caption could be something like, "Our mentor showed us some of the GH features that help track contributions." (This is an example from the [issues-and-projects](https://github.com/AgPipeline/issues-and-projects) repo.)
Screen Shot 2020-06-05 at 12 34 59 PM

**Comment by KristinaRiemer** [comment]
Your comment just made me realize that I never actually included Katrin's name in the post!

**Comment by KristinaRiemer** [comment]
Also can add link to OLS homepage with application for second cohort. https://openlifesci.org/

**Comment by KristinaRiemer** [comment]
Thanks for the comments @MagicMilly! I applied yours and a few others to the post. @dlebauer if you want to take a look and sign off on this?

**Comment by KristinaRiemer** [comment]
Considering this good enough to publish. Moving the content over to the old website and will publish officially on Monday.

**Pull request merged**